spec/javascripts/indigenousTerritoriesLayers_spec.js

```diff
@@ -0,0 +1,16 @@
+describe("Testing for indigenous Territories layer", function() {
+
+  beforeEach(function () {
+      loadFixtures('indigenousTerritoriesLayer.html');
+    });
+
+  it("Basic Test", function () {
+  expect(true).toBe(true);
+  });
+
+  it("Checks if layer polygons are present", function () {
+  // There should be atleast one polygon in the Territories layer always .
+  expect($("#map").children()[0].childNodes[2].childNodes[0].childNodes[0].childNodes.length).toBeGreaterThan(1);
```

Comment:
What I might do is to break this into different lines so we can narrow down what is undefined. So 
```suggestion
  expect($("#map")
    .children()[0]
    .childNodes[2]
    .childNodes[0]
    .childNodes[0]
    .childNodes.length)
    .toBeGreaterThan(1);
```

Maybe? Second, if you look at this example:

https://github.com/jywarren/pxlqst/blob/15887acea6bc297f1acd4785ad4f0ce400abfef4/spec/javascripts/pxlqst_spec.js#L85-L98

It does a pattern like `  you("walk west toward the torch and arrive there shortly after.", function(done) {`

then when the function is supposed to be complete, it calls `done()`. That allows us to peg completion of the test to something that may not complete immediately. 

Here is a pattern for simulating a remote call and running tests on the result:

https://github.com/publiclab/spectral-workbench.js/blob/eaee043e4588716a7b5a0855e5f0d6dbd1de2da1/spec/javascripts/spectrum_spec.js#L332-L367

It may also be helpful!